docs/PIPELINES_CONFIGURATION.md

```diff
@@ -0,0 +1,126 @@
+
+```
+kubectl patch tektonconfig config --type 'merge' --patch "$( cat <<EOF
+spec:
+  pipeline:
+    enable-bundles-resolver: true
+    enable-cluster-resolver: true
+    enable-custom-tasks: true
+    enable-git-resolver: true
+    enable-hub-resolver: true
+    enable-tekton-oci-bundles: true
+  chain:
+    artifacts.oci.storage: oci
+    artifacts.pipelinerun.format: in-toto
+    artifacts.pipelinerun.storage: oci
+    artifacts.taskrun.format: in-toto
+    artifacts.taskrun.storage: oci
+    transparency.enabled: "true"
+    transparency.url: http://rekor-server.\$NAMESPACE.svc
+EOF
+)"
+```
+
+8. Create the pipelines secret, containing your Github App's `Webhook URL` and `Webhook Secret`.
+
+```
+export APP_NAMESPACE=<your-app's namespace>
+export PIPELINES_SECRET_NAME=<your pipelines-secret name>
+export GITHUB_APP_WEBHOOK_SECRET=<your github app's webhook secret>
+export GITHUB_APP_WEBHOOK_URL=<your github app's webhook url>
+kubectl -n "$APP_NAMESPACE" create secret generic "$PIPELINES_SECRET_NAME" \
+    --from-literal="webhook-github-secret=$GITHUB_APP_WEBHOOK_SECRET" \
+    --from-literal="webhook-url=$GITHUB_APP_WEBHOOK_URL"
+```
+
+9. Create the `pipelines-as-code-secret`, containing your Github App's `App ID`, `Private Key`, `Webhook Secret`.
+
+```
+kubectl -n "$PIPELINES_NAMESPACE" create secret generic pipelines-as-code-secret \
+    --from-literal github-application-id="$GITHUB_APP_APP_ID" \
+    --from-literal github-private-key="$GITHUB_APP_PRIVATE_KEY" \
+    --from-literal webhook.secret="$GITHUB_APP_WEBHOOK_SECRET"
+```
+
+10. Fetch the codesign public key from the `signing-secrets` Secret inside the Operator's Namespace.
+
+```
+export COSIGN_SIGNING_PUBLIC_KEY=$(kubectl get secrets -n $PIPELINES_NAMESPACE signing-secrets -o jsonpath='{.data.cosign\.pub}')
+cat <<EOF | kubectl apply -f - >/dev/null
+apiVersion: v1
+data:
+    cosign.pub: $COSIGN_SIGNING_PUBLIC_KEY
+kind: Secret
+metadata:
+    labels:
+        app.kubernetes.io/instance: default
+        app.kubernetes.io/part-of: tekton-chains
+        operator.tekton.dev/operand-name: tektoncd-chains
+    name: cosign-pub
+    namespace: $APP_NAMESPACE
+type: Opaque
+EOF
+```
+
+11. Create the `pipelines-secret` in your application's Namespace, containing your Github App's `Webhook Secret`:
+
+```
+kubectl -n $APP_NAMESPACE create secret generic pipelines-secret --from-literal=webhook.secret=$GITHUB_APP_WEBHOOK_SECRET
+```
+
+12. Similarly with the previous step, create the `image-registry-token` in your application's Namespace, containing the docker `config.json` file of your Quay.io account (see more info [here](https://docs.redhat.com/en/documentation/red_hat_quay/3.6/html-single/use_red_hat_quay/index#allow-robot-access-user-repo)):
+
+```
+export IMAGE_REGISTRY_TOKEN_SECRET=<your-secret's-name>
+kubectl -n $APP_NAMESPACE create secret docker-registry "$IMAGE_REGISTRY_TOKEN_SECRET" --from-file=.dock
+erconfigjson=<your-docker-config.json-file-path>
```

Comment:
Copy pasting the code snippet in this line broke up the `.dock` from the `erconfigjson`, this should be on the same line as above.

Reply:
Oh ouch. I didn't notice that. Addressed here: [31ee0c4](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/19/commits/31ee0c4ea274efa813586fefd23cd975ee915374)